src/components/ResponsiveDrawer.js

```diff
@@ -149,7 +149,7 @@ class ResponsiveDrawer extends React.Component {
               <Text variant={['faded']}>
                 {this.props.network === '3'
                   ? 'running on Testnet'
-                  : 'please connect to Testnet'}
+                  : 'running on Mainnet'}
```

Comment:
`this.props.network === 1 ? 'running on Mainnet' : 'running on Testnet'`
makes more sense to me?

Reply:
k I can change that

Reply:
I'm gonna fix that on the prettify PR , otherwise there will be conflicts.. ok? @mjezew

Reply:
https://github.com/MainframeHQ/payments/pull/26/commits/3979da8b106180336c79a2bae612e4b779241ea9